lib/service/metric_client_service.dart

```diff
@@ -4,64 +4,114 @@ import 'dart:async';
 
 import 'package:autonomy_flutter/common/injector.dart';
 import 'package:autonomy_flutter/gateway/iap_api.dart';
+import 'package:autonomy_flutter/service/address_service.dart';
 import 'package:autonomy_flutter/service/configuration_service.dart';
+import 'package:autonomy_flutter/service/device_info_service.dart';
+import 'package:autonomy_flutter/util/metric_helper.dart';
+import 'package:sentry/sentry.dart';
 
 class MetricClientService {
   MetricClientService();
 
-  bool isFinishInit = false;
+  bool _isFinishInit = false;
```

Comment:
i don't see purpose of isFinishInit:
- the possibility other method call before initService is zero.
- set identifier is guarantee to success,
- no handle for case isFinishInit (eg: call initService)
- forget exist does not reset this